### DISCUSSION TITLE
API Full Access - Order and Inventory Management Apps not being approved.

### DISCUSSION THREAD

**Discussion by Gareth064** [etsy/open-api#610] [Q&A] [answered]
**Question**
Why are Etsy building an API with Endpoints to perform Order and Inventory Management if we aren't going to be able to get an App approved to use them?

No-one is going to need to use order and inventory management API's if all they have is one ETSY store. You'd just use the proper ETSY web app.

**Context**
So, most of are on here helping and contributing to the development of this new API, obviously because we are planning on using it!

I have been seeing lots of people in here raising issues, as well as on the Google Group, saying that their App isn't getting approved for full access.

Two days ago, someone in the Google group posted this: https://groups.google.com/g/etsy-api-v2/c/ckMkLqs_GTw/m/stpIN5RxBAAJ

![image](https://user-images.githubusercontent.com/22307223/193318980-a7248e0f-a206-4557-b93b-a9c80b7087fc.png)

**Multichannel Order and Inventory Management** is probably what 70%+ of developers are building apps for!! Including myself. Because most Etsy sellers don't want to have all their eggs in one basket so also sell on other Marketplaces. Which is just how you would run a handmade business if you want it to succeed.

I am a one-person developer, who in my spare time has spent close to a year building my app and to see this response from an Etsy dev has totally destroyed me.

**Comment by Gareth064** [comment]
I also reached out to the Etsy Dev's after posting this, and they are sticking to their guns of not approving new App Registrations in the Multichannel Inventory and Order Management category. I stressed how import this category is for genuine app developers who are trying to build apps to help small businesses in the Handmade and Antiques business.

The concern I think they have (and to be fair it's a concern I raised ages ago) is that developers are building apps which is allowing drop-shippers and resellers to flood the Etsy Marketplace with listings that aren't the type of listing that Etsy state the platform is to be used for. Like selling Chinese imported goods etc.

Not really sure how I am going to continue with my app, which fell into this category but was specific to helping crafty handmade type businesses. Maybe continue with it and hope that in the near future Etsy Developers figure out a way to better vet new Apps in the multichannel space.

@etsyachristensen @nateswart @capndesign - Is there anything you guys can put onto the Developers page\API Documentation to tell people that apps that are being built for this category are not going to be approved until further notice? It will stop a lot of new developers wasting their time building something, only to get to the point of going live and then finding out they can get the App Registration approved.

**Comment by vintagetwinmike** [comment]
I actually am trying to develop an app for just one Etsy shop and they won't approve it. I don't know what the point of having an API is if you  won't let anyone use it. What in the world are they looking for in order to approve people to use the stupid thing??

**Comment by nateswart** [comment, marked as answer]
There has been a large amount of working happening behind the scenes on this topic, with the (positive) impacts coming later this week. I would suggest holding off until next week, and then submitting again. As always, be sure you include as much information as possible in the key request form as we see _a lot_ of submissions come through with sample/incorrect/missing information which generates rejections.

**Reply by vintagetwinmike** [reply]
Thanks for that info. I'll try again next week. My request has been a pretty basic description of our intention. Ie, an app to sync our website backend to our etsy shop in order to list new products and delete products when they've sold on Etsy or elsewhere. That's literally all it has to do. Is that something that for some reason would be unacceptable?

**Reply by Gareth064** [reply]
> There has been a large amount of working happening behind the scenes on this topic, with the (positive) impacts coming later this week. I would suggest holding off until next week, and then submitting again. As always, be sure you include as much information as possible in the key request form as we see _a lot_ of submissions come through with sample/incorrect/missing information which generates rejections.

@nateswart have you got anything to share about the "positive impacts coming later this week", which would have been 2 weeks ago?

**Reply by etsyachristensen** [reply]
@Gareth064 We have begun approving applications that are multi-channel and inventory management again. The descriptions of the apps much be complete and descriptive of what is being requested. Also, be sure to include correct URLs.

**Reply by Gareth064** [reply]
@etsyachristensen brilliant news. I can start working on my app again. I gave up after hearing that multichannel apps were no longer being approved.
